Dockerfile

```diff
@@ -3,9 +3,10 @@ FROM alpine:3.9
 LABEL maintainer="technology@werkspot.com"
  
 RUN apk add --no-cache varnish
+RUN chown nobody:nogroup -R /etc/varnish
 
-ENV LISTEN_ADDRESS ":8080" \
-    WORKING_DIRECTORY="/tmp/varnish"
+ENV LISTEN_ADDRESS ":8080"
+ENV WORKING_DIRECTORY "/tmp/varnish"
```

Comment:
This one is fixed in the other PR, which is already merged

Reply:
:+1: